src/Stores/User/actions.js

```diff
@@ -0,0 +1,68 @@
+/* @flow */
+import { login, signUp } from '../../utils/api';
+
+import { saveToken } from '../../utils/tokenStorage';
+
+export const ActionTypes = {
```

Comment:
Thoughts on having the action types in a separate file? We do that at work and I did that in another PR

Reply:
I'm happy to have it like that.
Any preferences for filename? What's the PR? then I can just look it up :)

Reply:
`actionTypes.js`